File: sentimental-mario-less/mario.py
# TODO

from cs50 import get_int

while True:
    n = get_int("Height: ")
    if n > 0 and n < 9:
        break

for i in range(n):
    for j in range(n):
        if (n - j) > (i + 1):
            print(" ", end="")
        else:
            print("#", end="")
    print()

File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    do
    {
        n = get_int("Height = ");
    }
    while (n < 1 || n > 8);



    for (int f = 0; f < n; f++)
    {
        for (int c = 0; c < n; c++)
        {
            if (n - c > (f + 1))
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }

        }

        printf("\n");
    }


}